### ISSUE TITLE
After running overnight, I noticed a lot of escape pods all leaving the center of the system

### ISSUE THREAD

**Issue by smcameron** [smcameron/space-nerds-in-space#152]
Left snis_server running overnight, and in the morning, there were a lot of ships, distant from the center of the system (the "sun") and all travelling in different directions, but each one travelling away from the "sun" like a very large, very slow motion explosion made out of ships.  Also, all of these ships involved in the slow-motion explosion were of type SHIP_CLASS_ESCAPE_POD (== 21), which is pretty strange, because ...

```
scameron@wombat ~/github/space-nerds-in-space $ grep ESCAPE_POD *.c *.h
snis_ship_type.h:#define SHIP_CLASS_ESCAPE_POD 21
```

... there is no explicit code anywhere that mentions escape pods or that implements any escape pod specific behavior that I am aware of.  Escape pods are just another model, like any other, there's not really anything special about them. So the fact that they were all escape pods is an interesting and mysterious clue.

**Comment by smcameron** [comment]
Dumping out the brains of one of these escaping escape pods, I didn't figure out *why* it's doing what it's doing, but I did find that the brain was in PATROL mode, but it seemed to be heading in the wrong direction.

![escape-pod-dump](https://user-images.githubusercontent.com/23912/44957095-db776900-ae9b-11e8-9664-60ce93199f02.jpg)

Notice the current Z position is -168584, the current Z velocity is -4.33, and the destination Z is +58698. The Z velocity ought to be positive, not negative, if the guy is ever going to get to his destination. So he's going the wrong way.

**Comment by smcameron** [comment]
I notice the short term destination is not very far from the ship's current position.  I begin to suspect that this might be a bug in the planet avoidance code?  That is to say, the ship is trying to go around a (non-existent) planet perhaps?  But again.... why only escape pods????????????

**Comment by smcameron** [comment]
Last night, something happened which has only happened to me a few times during my life.  I debugged a program in my sleep.  I was dreaming about this bug, and I had the idea that escape pods don't warp, but maybe every other ship type does. Maybe the non-warp navigation is so broken that without warping a ship will never reach its destination. Could it be?

A quick debug patch...

```
@@ -19650,6 +19650,7 @@ static void override_asset_dir(void)
 static int read_ship_types(void)
 {
        char path[PATH_MAX];
+       int i;
 
        sprintf(path, "%s/%s", asset_dir, "ship_types.txt");
 
@@ -19660,6 +19661,9 @@ static int read_ship_types(void)
                        fprintf(stderr, "%s: %s\n", path, strerror(errno));
                return -1;
        }
+       for (i = 0; i < nshiptypes; i++)
+               fprintf(stderr, "%s warpchance = %d\n", ship_type[i].class, ship_type[i].warpchance);
+       exit(1);
        return 0;
 }
```

Produces this output:

CRUISER warpchance = 50
DESTROYER warpchance = 50
FREIGHTER warpchance = 3
TANKER warpchance = 5
TRANSPORT warpchance = 10
BATTLESTAR warpchance = 25
STARSHIP warpchance = 40
MINER warpchance = 2
SCIENCE warpchance = 5
SPACEFARER warpchance = 35
DRAGONHAWK warpchance = 20
SKORPIO warpchance = 50
DISRUPTOR warpchance = 50
RESEARCH warpchance = 3
CONQUEROR warpchance = 25
SCRAMBLER warpchance = 15
SWORDFISH warpchance = 15
WOMBAT warpchance = 50
DREADKNIGHT warpchance = 30
VANQUISHER warpchance = 60
ENFORCER warpchance = 50
ESCAPEPOD warpchance = 0
MANTIS warpchance = 50
```

Indeed, the escapepod is the only one with a warp chance of zero.

Taking a look at the navigation code, I begin to suspect the culprit is ai_add_ship_movement_variety().

So let's comment that out and see how things behave.

**Comment by smcameron** [comment]
After letting things run for awhile without the ai_add_ship_movement_variety() code, the escape pods no longer flee the system, and I didn't find ships heading in the wrong direction, away from their ostensible destination.

Fixed by 442fd166ae259eea29811202a6b21834504780cf
